Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Select.hs

```diff
@@ -120,6 +124,7 @@ select personId estimateId req@DEstimateSelectReq {..} = do
   estimate <- QEstimate.findById estimateId >>= fromMaybeM (EstimateDoesNotExist estimateId.getId)
   when (DEstimate.isCancelled estimate.status) $ throwError $ EstimateCancelled estimate.id.getId
   let searchRequestId = estimate.requestId
+  searchRetry <- SearchRetry.findById searchRequestId
```

Comment:
Because of this, one extra call will be added to DB. Select API itself can pass the parentSearchId.